packages/edit-widgets/src/components/header/style.scss

```
@@ -82,6 +82,7 @@
 	padding-right: $grid-unit-10;
 	padding-left: $grid-unit-20;
 	overflow: hidden;
+	height: 60px;
```

```suggestion
	height: $header-height;
```

The header height may change to 64px in the future, so it's a good idea to use a variable to prepare for that (See #60729).